tests/test_models_unet.py

```diff
@@ -133,6 +133,13 @@ def test_from_pretrained_hub(self):
 
         assert image is not None, "Make sure output is not None"
 
+    def test_from_pretrained_accelerate(self):
```

Comment:
Cool test! Could we maybe try to also test that:

a) loading with and without accelerate gives the same results when doing a forward pass
b) memory usage is reduced when using accelerate

Reply:
@patrickvonplaten just did it.